charcoal/Snakefile

```diff
@@ -287,21 +287,21 @@ rule contigs_sig:
     """
 
 # run a search, query.x.database.
-rule search_all:
+rule prefetch_all:
     input:
         query = stage1_dir + '/{filename}.sig',
         databases = config['gather_db']
     output:
         csv = stage1_dir + '/{filename}.matches.csv',
-        matches = stage1_dir + '/{filename}.matches.sig',
+        matches = stage1_dir + '/{filename}.matches.zip',
         txt = stage1_dir + '/{filename}.matches.txt'
     params:
         moltype = "--{}".format(config['moltype'].lower()),
         gather_scaled = config['gather_scaled']
     conda: 'conf/env-sourmash.yml'
     shell: """
-        sourmash search {input.query} {input.databases} -o {output.csv} \
-            --containment {params.moltype} --scaled {params.gather_scaled} \
+        sourmash gather --no-linear --prefetch {input.query} {input.databases} -o {output.csv} \
```

Comment:
I think this should be `sourmash prefetch` instead of `sourmash gather --prefetch`. The former outputs all the matches, the latter actually does a gather.

(`sourmash prefetch` is basically the same as `sourmash search --containment`, but uses different command line parameters, is faster, and also only works on scaled signatures.)